packages/astro-embed-link-preview/LinkPreview.astro

```diff
@@ -4,9 +4,11 @@ import { parseOpenGraph } from './lib';
 export interface Props {
 	/** URL to fetch Open Graph data. */
 	id: string;
+	/** Whether to show the video if supplied in OpenGraph */
+	hideMedia?: boolean;
 }
 
-const { id } = Astro.props;
+const { id, hideMedia = false } = Astro.props as Props;
```

Comment:
`as Props` hasn’t been needed for quite a while now! (I forget exactly when it changed.)

```suggestion
const { id, hideMedia = false } = Astro.props;
```

Reply:
Well, it's not needed for the external inference, but the props aren't autocompleted w/o it, my editor said `Astro.props` was typed as `any`.

I can adjust back to that if you want though

Reply:
Oh, interesting. Not the case for me:

<img width="442" alt="props in VS Code showing correct IntelliSense" src="https://github.com/delucis/astro-embed/assets/357379/1d5dabb2-ebb9-4257-b608-5dcf1775dca1">